source/index.ts

```diff
@@ -17,12 +19,20 @@ type TypedArray = Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint
 
 export interface Ow {
 	/**
-	 * Test if the value matches the predicate. Throws an `ArgumentError` if the test fails..
+	 * Test if the value matches the predicate. Throws an `ArgumentError` if the test fails.
 	 *
 	 * @param value Value to test.
 	 * @param predicate Predicate to test against.
 	 */
 	<T>(value: T, predicate: BasePredicate<T>): void;
+	/**
+	 * Test if `value` matches the provided `predicate`. Throws an `ArgumentError` with the specified `label` if the test fails.
+	 *
+	 * @param value Value to test.
```

Comment:
Unrelated, but in https://github.com/sindresorhus/typescript-definition-style-guide we enforce a dash between the paramater name and description. Do you think this makes sense or should we drop that?

Reply:
Yes I noticed that we didn't do it in this codebase. I'll create a new issue to fix that in the entire project in one PR.